src/sa_web/static/js/views/app-view.js

```diff
@@ -306,6 +318,25 @@ var Shareabouts = Shareabouts || {};
           attribute: 'properties',
 
           success: function() {
+            // generate an attribute with relevant story information
+            // NOTE: right now a single model can only be part of a single story.
+            collection.forEach(function(model, index) {
+              model.set("story", function() {
+                var storyObj = null,
+                url = model.attributes.datasetSlug + "/" + model.attributes.id;
+                _.each(self.options.storyConfig, function(story) {
+                  if (story.order[url]) {
+                    storyObj = {
+                      tagline: story.tagline,
+                      next: story.order[url].next,
+                      previous: story.order[url].previous
+                    }
+                  }
+                });
+                return storyObj;
+              }());
+            });
```

Comment:
re: moving the story logic to the model-- that's a good point. I don't see why that wouldn't work-- I'll refactor so we're adding story objects via `attributesToAdd`.